### PR TITLE
added links to open and ee uris.

### PR DESCRIPTION
**Pull request by orsenthil**
These are alpha builds. Useful to be part of the installer as we share it with the customers.
These can be removed once 1.13 is GA.

### CODE REVIEW

**Comment by bernadinm** [comment]
Thank you @orsenthil! I've gone and updated the changes directly. The only thing was missing was the single quotes as well as the DC/OS 1.13.0-alpha instructions 📖. 

I see that the links are currently not yet live at this time of writing. 

Apart from this, everything else LGTM!

File: ee/uri/download-variables.tf [hunk 1]
```diff
@@ -2,21 +2,22 @@ variable "dcos_download_path" {
...
+    "1.11.9"       = "https://downloads.mesosphere.com/dcos-enterprise/testing/1.11.9.1/commit/07200c92180d5749ed3f6bcbb0b69846a7cb1a6b/dcos_generate_config.ee.sh"
+    "1.11.10"      = "https://downloads.mesosphere.com/dcos-enterprise/testing/1.11.10/commit/07200c92180d5749ed3f6bcbb0b69846a7cb1a6b/dcos_generate_config.ee.sh"
+    "1.12.0"       = "https://downloads.mesosphere.com/dcos-enterprise/stable/commit/20fa047bbd37188ccb55f61ab9590edc809030ec/dcos_generate_config.ee.sh"
+    "1.12.1"       = "https://downloads.mesosphere.com/dcos-enterprise/stable/commit/b7f04138fbb43f81a157fbd1f64904681532f61e/dcos_generate_config.ee.sh"
+    "1.12.2"       = "https://downloads.mesosphere.com/dcos-enterprise/testing/1.12.2/commit/c3af9042b96c5f70ca304083d1ac760ccf82ad69/dcos_generate_config.ee.sh"
+    "1.12.3"       = "https://downloads.mesosphere.com/dcos-enterprise/stable/1.12.3/commit/e8ecb9c00dc5bdc27d830df0b7fc91b6311660b8/dcos_generate_config.ee.sh"
+    "1.13.0-alpha" = "http://downloads.mesosphere.com/dcos-enterprise/stable/1.13.0-alpha/commit/633f96c6b97ee6b48eb20cbd06f92cbfc01a6622/dcos_generate_config.sh"
```
Comment (orsenthil) [inline ee/uri/download-variables.tf]: This should be 
```
http://downloads.mesosphere.com/dcos-enterprise/stable/1.13.0-alpha/commit/4da3d45e19ce0c04193a237d5a170e0b08108305/dcos_generate_config.ee.sh
```

File: open/uri/download-variables.tf [hunk 2]
```diff
@@ -2,59 +2,60 @@ variable "dcos_download_path" {
...
+    "1.11.6"       = "https://downloads.dcos.io/dcos/stable/1.11.6/dcos_generate_config.sh"
+    "1.11.7"       = "https://downloads.dcos.io/dcos/stable/1.11.7/dcos_generate_config.sh"
+    "1.11.8"       = "https://downloads.dcos.io/dcos/stable/1.11.8/dcos_generate_config.sh"
+    "1.11.9"       = "https://downloads.dcos.io/dcos/stable/1.11.9/dcos_generate_config.sh"
+    "1.11.10"      = "https://downloads.dcos.io/dcos/stable/1.11.10/dcos_generate_config.sh"
+    "1.12.0"       = "https://downloads.dcos.io/dcos/stable/1.12.0/dcos_generate_config.sh"
+    "1.12.1"       = "https://downloads.dcos.io/dcos/stable/1.12.1/dcos_generate_config.sh"
+    "1.12.2"       = "https://downloads.dcos.io/dcos/stable/1.12.2/dcos_generate_config.sh"
+    "1.12.3"       = "https://downloads.dcos.io/dcos/stable/1.12.3/dcos_generate_config.sh"
+    "1.13.0-alpha" = "https://downloads.dcos.io/dcos/stable/1.13.0-alpha/commit/4da3d45e19ce0c04193a237d5a170e0b08108305/dcos_generate_config.ee.sh"
```
Comment (orsenthil) [inline open/uri/download-variables.tf]: This should be

```
https://downloads.dcos.io/dcos/stable/1.13.0-alpha/commit/633f96c6b97ee6b48eb20cbd06f92cbfc01a6622/dcos_generate_config.sh

```

**Review by orsenthil — COMMENTED** [review]
Hi @bernadinm  thank you. I think, I flipped the EE and OSS URI by mistake. I have provided the correct URIS

**Review by bernadinm — APPROVED** [review]

**Comment by bernadinm** [comment]
Thanks @orsenthil! I have updated the paths to what you've provided.

**Comment by orsenthil** [comment]
Thanks again. If you think we are good, let's merge this. I think, these will be useful if the dc/os customers want to try the alpha builds with the installer from the website.
// We wont' have any documentation for this, but new features of 1.13 can be tested.

Thank you!

**Pull request merged**